lesson_shell_fasta.md

```diff
@@ -0,0 +1,233 @@
+
+## Learning Objectives
+
+* Download files form FTP repositories
+* Working with compressed files
+* Working with FASTA files
+
+## Lesson
+
+In this lesson we will learn how to download data in [fasta format](https://en.wikipedia.org/wiki/FASTA_format) form the web and how to manipulate them using basic shell commands.
+We will use as sample data set the human [non-coding RNAs](https://en.wikipedia.org/wiki/Non-coding_RNA) (ncRNA).
+
+## Exercises
+### 1 - Downloading data
+
+The data we need are on the [Ensembl](http://www.ensembl.org/index.html) FTP site, a repository of all publicly available files from the project.
+
+There is an [html interface](http://www.ensembl.org/info/data/ftp/index.html) of the last release where it should be easy to locate the link to the FASTA file of the ncRNA
```

Comment:
I'm hesitant to use the words *should be easy* because it implies some level of ability on the participants part.  I would suggest just saying something like... 
In this interface, we need to locate the link to the FASTA file of human ncRNA on the page and click to download it.